packages/flutter/test/widgets/sliver_semantics_test.dart

```diff
@@ -151,34 +163,34 @@ void main() {
             tags: <SemanticsTag>[RenderViewport.useTwoPaneSemantics],
             children: <TestSemantics>[
               new TestSemantics(
-                id: 5,
+                id: 6,
                 actions: <SemanticsAction>[SemanticsAction.scrollUp, SemanticsAction.scrollDown],
                 children: <TestSemantics>[
                   new TestSemantics(
                     id: 2,
                     label: r'Item 0',
                     textDirection: TextDirection.ltr,
-                    nextNodeId: 3,
```

Comment:
Again, we need to find a way to assert that things are in the correct traversal order in a test.

Reply:
Done.